web/react/utils/client.jsx

```diff
@@ -279,24 +279,33 @@ module.exports.getAudits = function(userId, success, error) {
     });
 };
 
-module.exports.getMe = function(success, error) {
+module.exports.getMeSynchronous = function(success, error) {
+    var currentUser = null;
 
     $.ajax({
+        async: false,
```

Comment:
Ah okay cool had no idea, reading #370 made it seem like it was all uses of it and figured someone originally removed it here for the same reason.